runelite-client/src/main/java/net/runelite/client/plugins/chatnotifier/ChatNotifierConfig.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2018, Tanner <https://github.com/Reasel>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.chatnotifier;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "chatnotifier",
+	name = "Chat Notifier",
+	description = "Configuration for the chat notifier plugin"
+)
+public interface ChatNotifierConfig extends Config
+{
+	@ConfigItem(
+		keyName = "messageNotification",
+		name = "Notified messages",
+		description = "Configures specifically which messages to notify you of. Leave blank to disable. Format: message, message",
```

Comment:
It may also be useful to have an option to "notify highlight" or something/also an option to bold your name in chat, as this is requested occasionally.

Reply:
Wouldn't the bold name be better suited for something like the chat recolor plugin? I could make an option for notifying if you are mentioned, but I was holding off doing that since the player can simply put in their own name. Less need for toggles.

Also for the highlight notification I assumed that you only wanted to be notified by server messages instead of anyone saying a highlighted item...Meaning if you setup the valuable drop chat messages in-game then you can set your settings to notify you of highlighted drops. Also I am not sure this plugin is where that feature should be located. Wouldn't the Ground Items Plugin be an equally correct location and wouldn't need any cross plugin sharing of configs/functions.

Reply:
Wondering if I could get a follow up on this.